linker.go

```diff
@@ -272,9 +273,11 @@ func fixupAndValidate(insns asm.Instructions) error {
 	return nil
 }
 
-// POISON_CALL_KFUNC_BASE in libbpf.
-// https://github.com/libbpf/libbpf/blob/2778cbce609aa1e2747a69349f7f46a2f94f0522/src/libbpf.c#L5767
-const kfuncCallPoisonBase = 2002000000
+// A constant used to poison calls to non-existent kfuncs.
+//
+// Similar POISON_CALL_KFUNC_BASE in libbpf, except that we use a value lower
+// than 2^28 to fit into a tagged constant.
+const kfuncCallPoisonBase = 1<<28 - 1
```

Comment:
Would it make sense to export `platform.platformMask` for this purpose? This is basically `^platformMask`. I'm also not sure if all 1s retains its debuggability properties. We could also pick something else that fits in 28 bits like `0xdedc0de`, and it would be relevant, too. :wink:

Reply:
Went with 0xdedc0de, thanks!